Amazon.Ion.ObjectMapper.Test/Utils.cs

```diff
@@ -94,5 +96,35 @@ public static IIonValue StreamToIonValue(Stream stream)
         {
             return IonLoader.Default.Load(stream).GetElementAt(0);
         }
+        
+        public static IIonValue SerializeToIonWithCustomSerializer<T>(IonSerializer<T> customSerializer, T item)
+        {
+            return SerializeToIonWithCustomSerializers(
+                new Dictionary<Type, dynamic>() {{typeof(T), customSerializer}},
+                item);
+        }
+        
+        public static T DeserializeWithCustomSerializer<T>(IonSerializer<T> customSerializer, T item)
+        {
+            return DeserializeWithCustomSerializers(
+                new Dictionary<Type, dynamic>() {{typeof(T), customSerializer}},
+                item);
+        }
+        
+        public static IIonValue SerializeToIonWithCustomSerializers<T>(Dictionary<Type, dynamic> ionSerializers, T item)
```

Comment:
Can you update the `dynamic` here to be IonSerializer<T> as well? Same with below.

Reply:
That doesn't work because the `IonSerializers` option is explicitly expecting `Dictionary<Type, dynamic>`.